### PR TITLE
Dts 28092 new

### PR DESCRIPTION
**Pull request by eswbogol**

### CODE REVIEW

**Review by hargupta15 — DISMISSED** [review]

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/mongock/FieldMappingStructureForMongock.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package com.publicissapient.kpidashboard.apis.mongock;
```
[the remark this answers is not in the document]
Comment (eswbogol) [reply customapi/src/main/java/com/publicissapient/kpidashboard/apis/mongock/FieldMappingStructureForMongock.java:17]: done

**Review by shunaray — APPROVED** [review]

File: common/src/main/java/com/publicissapient/kpidashboard/common/model/application/KpiMaster.java [hunk 2]
```diff
@@ -57,8 +57,8 @@ public class KpiMaster extends BasicModel {
 	private String chartType;
 	private String upperThresholdBG;
 	private String lowerThresholdBG;
-	private String xAxisLabel;
-	private String yAxisLabel;
+	private String xaxisLabel;
```
Comment (hirbabar) [inline common/src/main/java/com/publicissapient/kpidashboard/common/model/application/KpiMaster.java:60]: have you changes on UI?

File: customapi/src/main/resources/json/mongock/default/metadata_identifier.json [hunk 3]
```diff
@@ -0,0 +1,1476 @@
+[
+  {
+    "tool" : "Jira" ,
+    "templateName" : "Standard Template" ,
+    "templateCode" : "7" ,
+    "isKanban" : false ,
```
Comment (hirbabar) [inline customapi/src/main/resources/json/mongock/default/metadata_identifier.json:6]: please verify all json file is same as master code

**Review by hirbabar — APPROVED** [review]

File: common/src/main/java/com/publicissapient/kpidashboard/common/model/application/KpiMaster.java (hunk 2 above)
Comment (eswbogol) [reply common/src/main/java/com/publicissapient/kpidashboard/common/model/application/KpiMaster.java:60]: no changes in UI, in UI also the same is used

**Review by hirbabar — APPROVED** [review]

**Review by shunaray — APPROVED** [review]

**Pull request merged**